src/View/Sidebar/BookmarkRow.vala

```diff
@@ -149,7 +149,6 @@ public class Sidebar.BookmarkRow : Gtk.ListBoxRow, SidebarItemInterface {
         label_stack.visible_child_name = "label";
 
         icon = new Gtk.Image.from_gicon (gicon, Gtk.IconSize.MENU);
-
```

Comment:
I don't love removing blank space between two different objects personally

Reply:
Well, these could be regarded as parts of one compound object (icon_label_grid) but in that case I should have removed the preceding blank line as well. I'll revert this change for now.